app/css/components/tooltips/concept.css

```diff
@@ -5,6 +5,8 @@
 
     @apply shadow-lg rounded-8 py-20 px-24 bg-white;
 
+    transition: opacity 120ms cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Could we comment this out (but leave it there) and put a `/* TODO: Add fade in */` here pls. I'm trying to keep all design stuff to only things that that Tai has approved for now.

Reply:
Sure

Reply:
Thank you for using the "correct" transition tho.